optuna/study.py

```diff
@@ -256,11 +259,18 @@ def optimize(
             n_jobs=1,  # type: int
             catch=(),  # type: Union[Tuple[()], Tuple[Type[Exception]]]
             callbacks=None,  # type: Optional[List[Callable[[Study, structs.FrozenTrial], None]]]
-            gc_after_trial=True  # type: bool
+            gc_after_trial=True,  # type: bool
+            show_progress_bar=False  # type: bool
     ):
         # type: (...) -> None
         """Optimize an objective function.
 
+        Optimization is done by choosing a suitable set of hyperparameter values from a given
+        range. Uses a sampler which implements the task of value suggestion based on a specified
+        distribution. The sampler is specified in :func:~optuna.study.create_study and the default
```

Comment:
Please make sure it's a proper link as follows. But I'm afraid you'll also have to break this line into two as it becomes too long.
```suggestion
        distribution. The sampler is specified in :func:`~optuna.study.create_study` and the default
```